CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ include(tests/arch_tests.cmake)
 include(tests/tests.cmake)
 
 add_custom_target(all-tests)
+add_custom_target(xilinx-timing-tests)
```

Comment:
we may call this target `all-timing-comparison-tests` or something like that to keep consistent with other `all` targets which are not bound to a specific arch/vendor